docsrc/hello_world.rst

```diff
@@ -82,44 +82,38 @@ which contains all messages written to the console.
 If the ``output_dir`` argument is omitted, the output files are written
 to a temporary directory which is deleted when the current Python session is terminated.
 
+.. code-block:: python
 
-Access the sample
-^^^^^^^^^^^^^^^^^
+    # specify data file
+    bernoulli_data = os.path.join(cmdstan_path(), 'examples', 'bernoulli', 'bernoulli.data.json')
 
-The :ref:`class_cmdstanmcmc` object stores the CmdStan config information and
-the names of the the per-chain output files.
-It manages and retrieves the sampler outputs as Python objects.
+    # fit the model 
+    bern_fit = bernoulli_model.sample(data=bernoulli_data, output_dir='.') 
 
-.. code-block:: python
+    # printing the object reports sampler commands, output files
+    print(bern_fit)
 
-   print(bern_fit)
 
-The resulting set of draws produced by the sampler is lazily instantiated
-as a 3-D ``numpy.ndarray`` (i.e., a multi-dimensional array)
-over all draws from all chains  arranged as draws X chains X columns.
-Instantiation happens the first time that any of the information
-in the posterior is accesed via properties:
-``draws``, ``metric``, or ``stepsize`` are accessed.
-At this point the stan-csv output files are read into memory.
-For large files this may take several seconds; for the example
-dataset, this should take less than a second.
+Access the sample
+^^^^^^^^^^^^^^^^^
 
-.. code-block:: python
+The :ref:`class_cmdstanmcmc` object
+provides properties and methods to access, summarize, and manage the sample and its metadata.
 
-    bern_fit.draws().shape
-    
-Python's index slicing operations can be used to access the information by chain.
-For example, to select all draws and all output columns from the first chain,
-we specify the chain index (2nd index dimension).  As arrays indexing starts at 0,
-the index '0' corresponds to the first chain in the :ref:`class_cmdstanmcmc`:
+The sampler and model outputs from each chain are written out to Stan CSV files.
+The CmdStanMCMC object assembles these outputs into a
+numpy.ndarray which contains all across all chains arranged as (draws, chains, columns). 
+The `draws` method returns the draws array.
+By default, it returns the underlying 3D array.
+The optional boolean argument ``concat_chains``, when ``True``,
+will flatten the chains resulting in a 2D array.
+the size of the draws dimension is the number of sampling iterations.
```

Comment:
'the size of the draws dimension is the number of sampling iterations.' seems like a dangling sentence to be deleted (and not true since thin is an option)

Reply:
good catch - removed.